src/components/DynamicFormContainer/DynamicForm/DynamicForm.tsx

```diff
@@ -70,7 +72,7 @@ export const DynamicFormRaw: FunctionComponent<DynamicFormProps> = ({
     setAttachments(nextAttachment);
   };
 
-  const uiSchema = data.formData.uiSchema || {};
+  uiSchema = uiSchema || {};
```

Comment:
Dont do param reassignment. This may result in unintended effects. Can simply do the defaults in the input itself.

https://stackoverflow.com/questions/23314806/setting-default-value-for-typescript-object-passed-as-argument

While it does not pose a problem by affecting the param passed in directly, if we make this of a habit then there could be bad things that can happen like:
![image](https://user-images.githubusercontent.com/7406870/95411445-0ffe5e80-0959-11eb-9d98-8c4138eadcf1.png)

Reply:
Oh wow that's interesting and TIL, thanks! Will fix

Reply:
fixed!